### PR TITLE
Add support for KHR_texture_transform to glTF loader

### LINKED ISSUE #4149: Add support for KHR_texture_transform to glTF loader
See https://github.com/KhronosGroup/glTF/pull/1015 for proposal.

This is the original implementation. It just needs a bit of clean up, naming updates, etc.
https://github.com/bghgary/Babylon.js/tree/EXT_texture_transform

### PR DESCRIPTION
**Pull request by bghgary**
Includes a new validation test:
image

Fixes #4149

### CODE REVIEW

**Comment by deltakosh** [comment]
Can you update the new what's new?:)

**Comment by bghgary** [comment]
Yup, was already doing it :)

File: src/Materials/Textures/babylon.texture.ts [hunk 1]
```diff
@@ -65,6 +65,24 @@
         @serialize()
         public wAng = 0;
 
+        /**
+         * Defines the center of rotation (U)
+         */
+        @serialize()
+        public uCenter = 0.5;
```
Comment (bghgary) [inline src/Materials/Textures/babylon.texture.ts]: Any opinions on the naming of this? It's just for rotation and not for scale. Should I call it uRotationCenter perhaps? It's a bit odd compared to uAng which is also for rotation.

Comment (deltakosh) [reply src/Materials/Textures/babylon.texture.ts]: I guess uRotationCenter is good.
uAng is OK as you cannot scale with an angle

Comment (bghgary) [reply src/Materials/Textures/babylon.texture.ts]: Done!

File: dist/preview release/what's new.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
 ## Major updates
 
 ## Updates
+- Add support for specifying the center of rotation to textures ([bghgary](http://www.github.com/bghgary))
+- Add support for KHR_texture_transform to glTF 2.0 loader ([bghgary](http://www.github.com/bghgary))
```
Comment (deltakosh) [inline dist/preview release/what's new.md]: Do you mind creating a gltf section like I did for previous what's new
I guess we will have more to come:)

**Comment by deltakosh** [comment]
Wonderful

**Pull request merged**